week-5/Homework/mandatory/3-project/js/main.js

```diff
@@ -0,0 +1,54 @@
+var blueBtn = document.getElementById("blueBtn");
+var orangeBtn = document.getElementById("orangeBtn");
+var greenBtn = document.getElementById("greenBtn");
+var jumbotron = document.querySelector(".jumbotron");
+var donateBtn = document.getElementById("donate");
+var volunteerBtn = document.getElementById("volunteer");
+
+blueBtn.addEventListener("click", function () {
+  jumbotron.style.backgroundColor = `#588fbd`;
+  donateBtn.style.backgroundColor = `#ffa500`;
+  volunteerBtn.style.backgroundColor = `black`;
+  volunteerBtn.style.color = `white`;
+});
+
+orangeBtn.addEventListener("click", function () {
+  jumbotron.style.backgroundColor = `#f0ad4e`;
+  donateBtn.style.backgroundColor = `#5751fd`;
+  volunteerBtn.style.backgroundColor = `#31b0d5`;
+  volunteerBtn.style.color = `white`;
+});
+greenBtn.addEventListener("click", function () {
+  jumbotron.style.backgroundColor = `#87ca8a`;
+  donateBtn.style.backgroundColor = `black`;
+  volunteerBtn.style.backgroundColor = `#8c9c08`;
```

Comment:
The purpose of this exercise was to query specific DOM elements without modifying anything in html file. You added `id` attributes to a few buttons which made this exercise slightly easier. Beside that, well done! All buttons work correctly🙂 However I can see some logic repetition here as all three event listeners make very similar changes. Can you think of the way how you could refactor the code so there is only one function responsible for making these changes? This is only something for you to think about, I'm not asking you to submit a refactored solution as this one is good enough🙂